lib/ui/page/feed/feedPage.dart

```diff
@@ -137,8 +123,18 @@ class _FeedPageBody extends StatelessWidget {
       child: SliverAppBar(
         floating: true,
         elevation: 0,
-        leading: _getUserAvatar(context),
-        title: customTitleText('Home'),
+        leading: Builder(
+          builder: (BuildContext context) {
+            return IconButton(
+              icon: const Icon(Icons.menu),
+              onPressed: () {
+                scaffoldKey.currentState.openDrawer();
+              },
+            );
+          },
+        ),
+        title: Image.asset('assets/images/icon-48.png'),
```

Comment:
@imsuriyan `icon-48.png` is small image hence it is pixelated. So use bigger twitter icon to avoid this.
``` dart
Image.asset('assets/images/icon-480.png', height: 40),
```

Reply:
Yaa! Right..
Icon-48.png looks blur when we added on appbar.